docs/OSD Stick Overlay.md

```diff
@@ -0,0 +1,31 @@
+# OSD Stick Overlay
+
+The OSD Stick Overlay displays the coordinates of your radio gimbals in the OSD.  These work like regular OSD elements and also support all 4 radio modes with mode 2 being the default.
+
+
+## Configuration
+
+Currently (Configurator 10.4.0) OSD Stick Overlays can only be configured via the CLI.  Layout your OSD using the configurator and save.  
```

Comment:
Instead of having this comment here, we should add support for positioning these elements to configurator - the data for it is already sent over MSP, since the entire `item_pos` array is sent. If we default the TX mode to 2, then this should cover it for most users.

Reply:
Ok, I'll remove all references to Configurator 10.4.0 and assume that the Configurator supports it.